templates/xcode.pkr.hcl

```diff
@@ -127,7 +127,9 @@ build {
     inline = [
       "source ~/.zprofile",
       "brew install graphicsmagick",
+      "brew install imagemagick",
       "brew install wix/brew/applesimutils",
+      "curl 'https://awscli.amazonaws.com/AWSCLIV2.pkg' -o 'AWSCLIV2.pkg' && sudo installer -pkg AWSCLIV2.pkg -target /"
```

Comment:
What do you think of using Homebrew and doing it in the `base` variant?

Reply:
While I could add it to the base variant I’m afraid that there is no homebrew support for AWSCLI

Reply:
Seems `brew install awscli` installs the same package. Here is a link to [the formula](https://formulae.brew.sh/formula/awscli).

Reply:
Oh, cool, thanks